### ISSUE TITLE
Is this project still under developing?

### ISSUE THREAD

**Issue by prikevs** [jbenet/go-libutp#3]
Hi, I have checked the project, I found that this project has not finished yet.
Will you continue to develop this lib?

Thanks.

**Comment by jbenet** [comment]
@prikevs yes, probably. But not in the next few weeks. We need to build github.com/libp2p/xtp first. Maybe a few weeks before we return to this.

**Comment by prikevs** [comment]
Ok. I'm going to build a P2P proxy tool relay on UDP with go. I will read the original libutp code and maybe I can contribute to this project.

**Comment by anacrolix** [comment]
I've got a completed wrapper [here](https://github.com/anacrolix/go-libutp).
